src/Mutator/FunctionSignature/PublicVisibility.php

```diff
@@ -24,16 +24,16 @@
     use InterfaceParentTrait;
 
     /**
-     * Replaces "public function..." with "protected function ..."
+     * Replaces "public function..." with "protected function ..." and "private function ..."
      *
      * @param Node $node
      *
-     * @return ClassMethod
+     * @return \Generator
      */
     public function mutate(Node $node)
     {
         /* @var ClassMethod $node */
-        return new ClassMethod(
+        yield new ClassMethod(
```

Comment:
What improvement adds the yield here instead of the return?

Reply:
this is a new feature (not yet released). It allows to return `N` mutations from one mutator. Note the second mutation on line 48 below.

Also, this is not the same as `reutnr [new ClassMethod(...), new ClassMethod(...)]`. Such code replaces 1 `Node` with an array of `Node`s, which is completely different behaviour.